src/LabLog.Domain/Events/RoomCreatedEvent.cs

```diff
@@ -4,6 +4,7 @@ namespace LabLog.Domain.Events
 {
     public class RoomCreatedEvent : IEventBody
     {
-        public string EventType => "RoomCreated";
+        public string EventType => EventTypeString;
+        public const string EventTypeString = "RoomCreated";

```

Comment:
Why not just have a single public const? Why create a public const, then reference it in a another variable? What's the purpose?

Reply:
The const is so u can use it statically, the member property so  I can ask the object who it is before casting it

Reply:
Given the const is pubilc, why can't you query that instead of using the member property?

Reply:
if I have an IEventBody (I don't know yet which actual event it is), then I can't query it statically. I have to ask the object what it's current value is, via the IEventBody interface. You cannot implement static values on interfaces.
So this basically allows us to understand from an unknown event what type it is, but also to have a oconstant value to avoid "magic strings"